recipe/build.sh

```diff
@@ -1,19 +1,32 @@
 #!/bin/bash
+# Get an updated config.sub and config.guess
+cp $BUILD_PREFIX/share/gnuconfig/config.* ./config
+
+if [[ "${CONDA_BUILD_CROSS_COMPILATION}" == "1" ]]; then
+  # See https://github.com/tbeu/matio/issues/78#issuecomment-357797381
+  MATIO_CROSS_COMPILATION_CONFIGURE_OPTS="ac_cv_va_copy=C99"
+else
+  MATIO_CROSS_COMPILATION_CONFIGURE_OPTS=""
+fi
+
+
 
 ./configure \
   --enable-mat73 \
   --with-pic \
   --enable-extended-sparse \
   --prefix="${PREFIX}" \
   --with-zlib="${PREFIX}" \
-  --with-hdf5="${PREFIX}"
+  --with-hdf5="${PREFIX}" ${MATIO_CROSS_COMPILATION_CONFIGURE_OPTS}
 make
 
+if [[ "${CONDA_BUILD_CROSS_COMPILATION}" != "1" ]]; then
```

Comment:
I don't understand how the latest non-`osx-arm64` builds did not fail, whereas for other feedstocks this line triggers `unbound variable` errors, like https://github.com/conda-forge/libexif-feedstock/pull/1. Is it because `CONDA_BUILD_CROSS_COMPILATION` was referenced above?

Reply:
I think it is the line
```
set -euo pipefail
```

Reply:
Ah, I think you're right. I just learned about these options, e.g. that it is the `-u` that makes the shell exit when it tries to expand an unset variable. Should that be set here as well?

Reply:
no
it was probably set there fore debugging purposes. If the package builds, keep it simple.